database/rssfeeds.sql

```diff
@@ -93,33 +92,33 @@ CREATE TABLE `news` (
 -- Indices de la tabla `categories`
 --
 ALTER TABLE `categories`
-  ADD PRIMARY KEY (`categoryId`);
+  ADD PRIMARY KEY (`id`);
 
 --
 -- Indices de la tabla `categories-feeds`
 --
 ALTER TABLE `categories-feeds`
-  ADD PRIMARY KEY (`feedId`,`noCategory`,`categoryId`),
+  ADD PRIMARY KEY (`feedId`,`categoryId`) USING BTREE,
```

Comment:
Creo que MySQL por [default usa esta sentencia](https://stackoverflow.com/questions/1687910/advantage-of-btree)
```suggestion
  ADD PRIMARY KEY (`feedId`,`categoryId`),
```